packages/web-components/fast-foundation/src/data-grid/data-grid.ts

```diff
@@ -19,11 +19,21 @@ import {
     keyPageUp,
 } from "@microsoft/fast-web-utilities";
 import type { FASTDataGridCell } from "./data-grid-cell.js";
-import type { FASTDataGridRow } from "./data-grid-row.js";
+import type {
+    DataGridRowSelectionChangedDetail,
+    FASTDataGridRow,
+} from "./data-grid-row.js";
 import { DataGridRowTypes, GenerateHeaderOptions } from "./data-grid.options.js";
 
 export { DataGridRowTypes, GenerateHeaderOptions };
 
+/**
+ * Describes the how the dialog element handles selection
+ *
+ * @public
+ */
+export type DataGridSelectionMode = "none" | "single-row" | "multi-row";
```

Comment:
Should we have a single type as a const object that could be shared between DG Row and DG?

Reply:
Rows currently don't have a notion of "SelectionMode" , but moved the type definition to the options file so it could be shared if necessary.